plugins/evernote.koplugin/main.lua

```diff
@@ -17,6 +17,7 @@ local N_ = _.ngettext
 local slt2 = require('slt2')
 local MyClipping = require("clip")
 local realpath = require("ffi/util").realpath
+local json = require("json")
```

Comment:
Nit about order (see coding style). Clearly this plugin was never updated with them though so not a big deal for this PR.